src/main/java/uk/gov/register/resources/HttpServletResponseAdapter.java

```diff
@@ -5,25 +5,44 @@
 
 import javax.servlet.http.HttpServletResponse;
 import javax.ws.rs.core.HttpHeaders;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
 
-class HttpServletResponseAdapter {
+public class HttpServletResponseAdapter {
     private final HttpServletResponse httpServletResponse;
 
-    HttpServletResponseAdapter(HttpServletResponse httpServletResponse) {
+    public HttpServletResponseAdapter(HttpServletResponse httpServletResponse) {
         this.httpServletResponse = httpServletResponse;
     }
 
-    void addInlineContentDispositionHeader(String fileName) {
+    public void addInlineContentDispositionHeader(String fileName) {
         ContentDisposition contentDisposition = ContentDisposition.type("inline").fileName(fileName).build();
         httpServletResponse.addHeader(HttpHeaders.CONTENT_DISPOSITION, contentDisposition.toString());
     }
 
-    void addLinkHeader(String rel, String value) {
+    public void addLinkHeader(String rel, String value) {
```

Comment:
Wonder whether it's worth renaming all the methods here to be `set...` rather than `add...` to be consistent and not mislead?